sympy/stats/crv_types.py

```diff
@@ -2449,6 +2451,93 @@ def LogLogistic(name, alpha, beta):
 
     return rv(name, LogLogisticDistribution, (alpha, beta))
 
+#-------------------------------------------------------------------------------
+#Logit-Normal distribution------------------------------------------------------
+
+class LogitNormalDistribution(SingleContinuousDistribution):
+    _argnames = ('mu', 's')
+    set = Interval(0, 1)
+
+    @staticmethod
+    def check(mu, s):
+        _value_check(s**2 > 0, "Scale parameter s must be positive.")
+
+    def logit(self, x):
+        return log(x / (1 - x))
+
+    def pdf(self, x):
+        mu, s = self.mu, self.s
+        return exp(-(self.logit(x) - mu)**2/(2*s**2))*(S.One/sqrt(2*pi*(s**2)))*(1/(x*(1 - x)))
+
+    def _cdf(self, x):
+        mu, s = self.mu, self.s
+        return (S.One/2)*(1 + erf((self.logit(x) - mu)/(sqrt(2*s**2))))
+
+    def _moment_generating_function(self, t):
+        print("No analytical solution")
```

Comment:
```suggestion
```
If no analytical solution is available, do not define that method in the subclass.